docs/documentation/release_notes/topics/26_4_0.adoc

```diff
@@ -115,44 +168,42 @@ The {project_name} Operator now deploys {project_name} across multiple availabil
-ifeval::[{project_community}==true]
-Thank you to https://github.com/tnorimat[@tnorimat] for contributing this.
-endif::[]
+Set the new option `http-management-scheme` to `http` to have the management interface use HTTP rather than inheriting the HTTPS settings of the main interface.
+This allows monitoring those endpoints in environments where no TLS client is available.
 
-= Automatic certificate management for SAML clients
+== Expose health endpoints on the main HTTP(S) port
 
-The SAML clients can now be configured to automatically download the signing and encrypting certificates from the SP entity metadata descriptor endpoint. In order to use this new feature, in the client *Settings* tab, section *Signature and Encryption*, configure the *Metadata descriptor URL* option (the URL where the SP metadata information with the certificates is published) and activate *Use metadata descriptor URL*. The certificates will be automatically downloaded and cached in the `public-key-storage` SPI from that URL.
+With `health-enabled` set to true, you may set the `http-management-health-enabled` to `false` to indicate that health endpoints should be exposed on the main HTTP(s) port instead of the
+management port. When this option is `false` you should block unwanted external traffic to `/health` at your proxy.
 
-For more information, see link:{adminguide_link}#_client-saml-configuration[Creating a SAML client] in the {adminguide_name}.
+This allows using the health endpoints in environments where the load balancer might need access to those ports to direct traffic to the correct nodes.
 
-= Operator creates a ServiceMonitor automatically
+== Specify a `tlsSecret` on the Keycloak CR `ingress` spec
 
-The Operator now provisions a `ServiceMonitor` for the management endpoint if metrics are enabled and the
-`monitoring.coreos.com/v1:ServiceMonitor` Custom Resource Definition is present on the Kubernetes cluster. The
-specification of the `ServiceMonitor` takes into account the various management endpoint configurations, to ensure that
-metrics can be scraped without any additional configuration. If you do not want a `ServiceMonitor` to be created, you can disable
-this by setting `spec.serviceMonitor.enabled: false`. For more details, see the link:{operatorguide_link}[{operatorguide_name}].
+To support basic TLS termination (edge) deployments by the operator, you may now set the Keycloak CR `spec.ingress.tlsSecret` field to a TLS Secret name in the namespace.
+
+ifeval::[{project_community}==true]
+////
+Do not show this in the product release notes as on Kubernetes/OpenShift it is still
+simpler to use the Cache CRs as it will provide a smoother first startup experience.
+When using multiple sites, the first start of Keycloak will create the caches and then fail
+until Keycloak starts up on the second site and creates the caches there as well.
+////
```

Comment:
`cache-remote-backup-sites` I don't really follow what it does, but doesn't seem to have anything to do with remote caches being automatically created?

Reply:
Yes, it does: when creating a remote cache, it is also necessary to specify the backup sites for a multi-site setup.